src/AWS.Deploy.Recipes/CdkTemplates/BlazorWasm/BlazorWasm.csproj

```diff
@@ -27,9 +27,11 @@
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.107.0" />
-    <PackageReference Include="Amazon.CDK.AWS.S3" Version="1.107.0" />
-    <PackageReference Include="Amazon.CDK.AWS.S3.Deployment" Version="1.107.0" />
+    <PackageReference Include="Amazon.CDK" Version="1.121.0" />
```

Comment:
Updated to the latest CDK. I'll update the CDK reference on the other templates as I go through them in later PRs.